primitives/src/const_vec.rs

```diff
@@ -0,0 +1,99 @@
+//! An implementation of compact container for constant bytes.
+#[cfg(not(feature = "std"))]
+use alloc::{boxed::Box, format, string::String, vec::Vec};
+use core::ops::Deref;
+#[cfg(feature = "std")]
+use std::{boxed::Box, vec::Vec};
+
+use iroha_schema::{IntoSchema, MetaMap, Metadata, TypeId, VecMeta};
+use parity_scale_codec::{WrapperTypeDecode, WrapperTypeEncode};
+use serde::{Deserialize, Serialize};
+
+/// Stores bytes that are not supposed to change during the runtime of the program in a compact way
```

Comment:
well technically, it could possible to mutate the `ConstVec`, just not to resize it. But we don't need that